examples/pbr.py

```diff
@@ -13,39 +13,25 @@
 
 
 def load_gltf(path):
```

Comment:
We could extract this function from the example and put it alongside trimesh_mesh. It's a pretty useful thing to have a gltdf loader, even if it's not entirely feature complete just yet

Reply:
Good idea!